docs/docs/examples/frontend.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Frontend
+---
+
+
+### Scope
+
+To make our examples more accessible and user-friendly, we've outlined the capabilities that each example includes.
+By consulting the capabilities list, you can quickly determine whether a particular example has the features you need.
+
+As a general rule, we only include examples that meet the capabilities listed on our site.
+If we do have an SDK available for a particular example, we'll use it.
+Otherwise, we rely on existing authentication libraries.
+
+**Authentication with ZITADEL**
+
+A user should be able to login with ZITADEL.
+The flows that are used can differ depending on the programming language and client type.
+
+**Session persistence**
+
+The session of the user is cached.
+If you like to enable caching for your application depends highly on the use case and the security requirements you have.
+
+**Authorization  / Roles**
+
+In the example two different user types with different roles will be used.
+
+"User": Show the authorizations the user has himself
+
+"Admin": Access to a private route/page that is only accessible with the Admin role
+
+**Loading User Profile**
+
+The example shows the information of the authenticated user by requesting the userinfo endpoint.
+
+**Session Management**
+
+Depending on the authentication flow the example shows you how to handle refresh tokens (mobile/server-side) and silent refresh/silent sign-in (SPA).Scope
+Good to know if you like to use silent refresh/sign in: [Enable iFrame embedding](../guides/solution-scenarios/configurations#enable-iframe-embedding)
+
+**Logout**
+
+We show you how to use the end_session endpoint to handle a client initiated logout of a user and how to terminate a local session.
+Read more about the different possibilities of logout in our [guide](../guides/integrate/logout)
+
+### Single Page Application
+
+<table>
+  <tr>
+    <th></th>
+    <th></th>
+    <th>Auth</th>
+    <th>Session Persistence</th>
+    <th>Roles</th>
+    <th>User Profile</th>
+    <th>Session Management</th>
+    <th>Logout</th>
```

Comment:
I think we should not show this information, but should use it as internal guideline.